source/backends/debian/debpkgindex.d

```diff
@@ -195,7 +200,7 @@ public:
             if (!name)
                 continue;
 
-            auto pkg = new DebPackage (name, ver, arch);
+            auto pkg = newPackage (name, ver, arch);
```

Comment:
I don't like having to name the classes in the code like that, so I prefer to leave it as is unless there's a way around that. It's nice that the Debian backend doesn't mention the Ubuntu one at all currently.

I don't think the `static if` bit would be required in this case, though. There's no Ubuntu specific stuff at this level.